api/src/main/java/javax/enterprise/inject/spi/BeanAttributes.java

```diff
@@ -67,6 +67,8 @@
 
     /**
      * Determines if the bean is an {@linkplain javax.enterprise.inject.Alternative alternative}.
+     *
+     * If it is an alternative it may may implement {@link Prioritized} to be selected for the entire application with a priority value.
```

Comment:
I put it here because it's the only place where we talk about alternative custom bean.
I agree with Martin and take his wording proposal.